src/main/java/org/jenkinsci/plugins/githubautostatus/notifiers/StatsdNotifier.java

```diff
@@ -76,8 +76,7 @@ private String getBranchPath() {
      * @param buildState the reported state
      */
     public void notifyBuildState(String jobName, String nodeName, BuildState buildState) {
-        String fqp = String.format("%s.stage.%s.status.%s", getBranchPath(), nodeName, buildState);  
-        client.increment(fqp, 1);
+        notifyBuildStageStatus(jobName, nodeName, buildState, 0);
```

Comment:
if there is no additional processing done in `notifyBuildState`, is it still necessary to keep it?

Reply:
It's called by the graphlistener itself and used in the influx side of things, don't think we can remove without re-writing the influx side as well.